mtags/src/main/scala-3/scala/meta/internal/pc/ScalaPresentationCompiler.scala

```diff
@@ -93,6 +93,7 @@ case class ScalaPresentationCompiler(
     workspace: Option[Path] = None
 ) extends PresentationCompiler:
 
+  //I should see name + age in autocomplete
```

Comment:
Is this just leftover or can we put more context for what this is referring to?

Reply:
leftover, I was testing out the other issue and forgot to remove part of the tested code :sweat: